.travis.yml

```diff
@@ -16,6 +18,15 @@ python:
 - '3.6'
 matrix:
   include:
+  - os: osx
+    language: generic
+    env: NMODE=nikola
+    python: '3.6'
+    before_install:
+    - brew upgrade pyenv
```

Comment:
You could argue the same about Travis CI’s images (in fact, I had to take down a C project of mine because Travis introduced random crashes I can’t debug in any way).

I’m going to merge this and experiment using Python from Homebrew. (and if it breaks, I’ll be the first to know as that’s how I’m working on my Mac).